### PR TITLE
#55993 - upgrade to netstandard 2.1

### PR DESCRIPTION
**Pull request by tabman83**

### CODE REVIEW

File: src/Eshopworld.Tests.Core.Analyzers/Eshopworld.Tests.Core.Analyzers.csproj [hunk 1]
```diff
@@ -13,6 +13,8 @@
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <TargetFrameworkProfile>Profile7</TargetFrameworkProfile>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <NuGetPackageImportStamp>
```
Comment (micszym) [inline src/Eshopworld.Tests.Core.Analyzers/Eshopworld.Tests.Core.Analyzers.csproj]: unnecessary?

File: src/Tests/Eshopworld.Tests.Core.Analyzers.Test/Eshopworld.Tests.Core.Analyzers.Test.csproj [hunk 2]
```diff
@@ -13,6 +14,8 @@
     <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
+    <NuGetPackageImportStamp>
```
Comment (micszym) [inline src/Tests/Eshopworld.Tests.Core.Analyzers.Test/Eshopworld.Tests.Core.Analyzers.Test.csproj]: unnecessary?

**Review by micszym — APPROVED** [review]

File: src/Eshopworld.Tests.Core.Analyzers/Eshopworld.Tests.Core.Analyzers.csproj (hunk 1 above)
Comment (tabman83) [reply src/Eshopworld.Tests.Core.Analyzers/Eshopworld.Tests.Core.Analyzers.csproj]: tbh, no idea how it got added

File: src/Tests/Eshopworld.Tests.Core.Analyzers.Test/Eshopworld.Tests.Core.Analyzers.Test.csproj (hunk 2 above)
Comment (tabman83) [reply src/Tests/Eshopworld.Tests.Core.Analyzers.Test/Eshopworld.Tests.Core.Analyzers.Test.csproj]: looks like it's safe to remove
https://stackoverflow.com/a/24055340

**Review by micszym — APPROVED** [review]

File: src/Eshopworld.Tests.Core/Eshopworld.Tests.Core.csproj [hunk 3]
```diff
@@ -14,19 +14,19 @@
...
     <PackageTags>Test, Unit Test, xUnit, Moq, FluentAssertions</PackageTags>
     <AssemblyName>Eshopworld.Tests.Core</AssemblyName>
     <RootNamespace>Eshopworld.Tests.Core</RootNamespace>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageReleaseNotes>added Level3 tests</PackageReleaseNotes>
+    <PackageReleaseNotes>Upgraded to netstandard2.1</PackageReleaseNotes>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Eshopworld.Tests.Core.Analyzers" Version="1.0.5" />
```
Comment (micszym) [inline src/Eshopworld.Tests.Core/Eshopworld.Tests.Core.csproj:26]: Looks like we will need to update this in two steps, analyzers being a seperate one

**Pull request merged**